homeassistant/components/yale/const.py

```diff
@@ -0,0 +1,49 @@
+"""Constants for Yale devices."""
+
+from yalexs.const import Brand
+
+from homeassistant.const import Platform
+
+DEFAULT_TIMEOUT = 25
+
+CONF_ACCESS_TOKEN_CACHE_FILE = "access_token_cache_file"
+CONF_BRAND = "brand"
+CONF_LOGIN_METHOD = "login_method"
+CONF_INSTALL_ID = "install_id"
+
+VERIFICATION_CODE_KEY = "verification_code"
+
+DEFAULT_BRAND = Brand.YALE_HOME
+
+MANUFACTURER = "Yale Home Inc."
+
+DEFAULT_AUGUST_CONFIG_FILE = ".yale.conf"
```

Comment:
This doesn't seem used.

Reply:
https://github.com/home-assistant/core/pull/122804

Reply:
64a48e9cca723cc5159b5d467d827ee40bc5eddd